src/lib/createables.ts

```diff
@@ -676,6 +679,20 @@ const Createables: Createable[] = [
 			"Ava's assembler": 1
 		}),
 		QPRequired: 205
+	},
+	{
+		name: 'Dragon full shield',
```

Comment:
```suggestion
		name: 'Dragon sq shield',
```
give it the name of the outputted item, its confusing to a user to have to type full shield when they want a square shield